ethcore/res/ethereum/eip161_test.json

```diff
@@ -24,7 +24,7 @@
 		"eip161dTransition": "0x0",
 		"eip98Transition": "0x7fffffffffffffff",
 		"eip86Transition": "0x7fffffffffffffff",
-		"eip155Transition": "0x7fffffffffffffff",
+		"eip155Transition": "0x0",
```

Comment:
Setting to `0` enables the feature right?

Reply:
Yes, otherwise some transaction tests for `EIP158` spec (which for us is actaully eip161_test.json) fail, cause they assume that eip155 is enabled as well.